man/discordo.1

```diff
@@ -0,0 +1,70 @@
+.nh
+.TH discordo 1
+.SH NAME
+.PP
+\fB\fCdiscordo\fR - A lightweight, secure, and feature-rich Discord terminal client
+
+.SH SYNOPSIS
+.PP
+\fB\fCdiscordo\fR [\fB\fC--token "TOKEN"\fR]
+
+.SH DESCRIPTION
+.PP
+\fB\fCdiscordo\fR is a lightweight, secure, and feature-rich Discord terminal client.
+
+.PP
+If no token is provided, you will be prompted with an interactive login.
+.br
+The token is stored securely in the default OS-specific keyring.
+
+.SH USAGE
+.PP
+The default keybindings in the app are as follow
+
+.SS Guild tree
+.TS
+allbox;
+l l 
+l l .
+\fB\fCAction\fR	\fB\fCKeybinding\fR
+Focus	Alt + g
+.TE
+
+.SS Message text
```

Comment:
It is called "Messages Text". Similarly, "Guilds Tree" and "Message Input"

Reply:
Fixed it